precompiles/common/src/lib.rs

```diff
@@ -14,16 +14,8 @@
 // You should have received a copy of the GNU General Public License
 // along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
 
-use core::marker::PhantomData;
+#![cfg_attr(not(feature = "std"), no_std)]
 
-pub struct Precompile<R>(PhantomData<R>);
-
-#[precompile_utils_macro::precompile]
-impl<R> Precompile<R> {
-	#[precompile::public("foo()")]
-	fn foo() {
-		todo!()
-	}
-}
-
-fn main() { }
\ No newline at end of file
+/// System account size in bytes = Pallet_Name_Hash (16) + Storage_name_hash (16) +
+/// Blake2_128Concat (16) + AccountId (20) + AccountInfo (4 + 12 + AccountData (4* 16)) = 148
+pub const SYSTEM_ACCOUNT_SIZE: u64 = 148;
```

Comment:
Can you move this constant in crate primitives/account instead?
This way we can remove the precompile-common crate